### ISSUE TITLE
Operation FCOP

### ISSUE THREAD

**Issue by NTaylorMullen** [dotnet/aspnetcore#23320]
Operation **Fresh Coat of Paint** represents our effort to re-do the colors of the Razor/Blazor editors to better appeal to todays developers.

Current goals:

1. Color blind accessible
1. Proper levels of contrast between languages
1. ....

**Comment by NTaylorMullen** [comment]
@danroth27 were you able to follow up here to validate our "Razor transition" colorization choice?

**Comment by danroth27** [comment]
From the Razor syntax coloring surveys, we hear strong support for not having background coloring in Razor, but there are still a minority of Razor users that prefer the background coloring. We should ensure that users that prefer the current coloring have the option to turn background coloring back in Visual Studio.

**Comment by ryanbrandenburg** [comment]
Waiting on a WebTools insertion into VS before we can merge the VSLanguageServerClient PR.

**Comment by ryanbrandenburg** [comment]
WebTools and VSLanguageServerClient are in! Now just waiting for VSLanguageServerClient to go into VS feature/d17initial. That might be a bit slow given the other work going on, but it shouldn't require any extra input from us.

**Comment by NTaylorMullen** [comment]
Done?

**Comment by ryanbrandenburg** [comment]
Yeah, we can consider anything else a new issue.
